docs/source/validate.md

```diff
@@ -214,11 +214,11 @@ X = hcat([4., 5.] .+ 0.4 * randn(2, 10),
          [9., -5.] .+ 0.4 * randn(2, 5),
          [-4., -9.] .+ 1 * randn(2, 5))
 
-nclusters = 2:5
-clusterings = kmeans.(Ref(X), nclusters)
+№_clu = 2:5
```

Comment:
I think "№" symbol [typically denotes "sequential number"](https://en.wikipedia.org/wiki/Numero_sign), e.g. "cluster №3", but not the quantity of items.

Reply:
I am no expert here, I always thought it is just old abbreviation for "number" with no further nuance. We can change to `n_clu` or something similar.

Reply:
I reverted it back to *nclusters*